scripts/port2alias

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import subprocess
+from portconfig import get_port_config
+
+
+def get_platform_hwsku():
+    hwsku = None
+    platform = None
+    command = "show platform summary"
+    p = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
+    output = p.stdout.readlines()
+    for line in output:
+        tokens = line.split()
+        if not tokens:
+            continue
+        if tokens[0].lower() == 'hwsku:':
+            hwsku = tokens[1]
+        elif tokens[0].lower() == 'platform:':
+            platform = tokens[1]
+    return (platform, hwsku)
+
+def main():
```

Comment:
it is probably good to have a unit test for this string conversion. We need to have some positive and negative case.

for example,

Ethernet3 should not be converted if Ethernet3 is not in the alias map.

Ethernet43 should not be converted if Etherent4 is in but not Ethernet43,

Ethernet4: should be converted if Ethernet4 is in.